srv/components/controller/files/scripts/provision.sh

```diff
@@ -1455,21 +1496,28 @@ fw_update()
             echo "Getting the progress on fw update, please wait..." | tee -a $logfile
             printf '%s\n' "lcd $tmpdir" "get progress" > $tmpdir/progress.bf
```

Comment:
This is a bit cryptic. Could you document a comment specifying how the command works and what outcome is expected?

Reply:
done.